modules/codemod/lib/v5/getImportRenameMap.ts

```diff
@@ -1,4 +1,4 @@
-import {Collection, ImportDeclaration, JSCodeshift, CallExpression} from 'jscodeshift';
```

Comment:
Did the codemod's need to get touched in this PR?

Reply:
Sort of. `jscodeshift` was updated from `0.11.0` to `0.13.1` to remove the `colors` dependency. This included a change to the type definitions of `find` which now requires a `boolean` return. This effected almost all the v6 codemods. This forced me to review the logic in the codemod. Upon inspection I noticed 90% of the codemods were the same. I had 2 options:

1. Add `boolean` returns to all instances of `root.find`
2. Create a `renameImports` utility with the changes and update codemods to use it thus removing a lot of redundant code

I chose 2️⃣ because it helps to understand what the codemods do. It also helps when more need to be created.